webapp/channels/src/components/linking_landing_page/linking_landing_page.tsx

```diff
@@ -79,6 +79,11 @@ export default class LinkingLandingPage extends PureComponent<Props, State> {
         return landingPreference && landingPreference === LandingPreferenceTypes.BROWSER;
     };
 
+    isEmbedded = () => {
+        // this cookie is set when Mattermost is embedded in another app via iframe.
+        return document.cookie.includes('MMEMBED=1');
```

Comment:
This check is bad. It matches cookies like `whatever=MMEMBED=1`.

Reply:
Fixed.  Fully parsing the cookie now. Tested for `whatever=MMEMBED=1` which correctly returns false.